### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by cjellick**
Add docs to site for quickstart, airgap install, and policy management.

### CODE REVIEW

**Review by brandond — APPROVED** [review]

File: docs/policies.md [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+Creation and application of the PSPs is controlled by the presence or absence of certain annotations on the `kube-system` namespace. These map directly to the PSPs which can be created and are:
+
+ * `psp.rke2.io/global-restricted`
+ * `psp.rke2.io/system-unrestricted`
+ * `psp.rke2.io/global-unrestricted`
+
+The following logic is performed at startup for the policies and their annotations:
+
+* If the annotation exists, RKE2 continues without further action
+* If the annotations doesn't exist, RKE2 checks to see if the associated policy exists and if so, deletes and recreates it, along with adding the annotation to the namespace.
```
Comment (briandowns) [inline docs/policies.md:23]:
```suggestion
* If the annotation doesn't exist, RKE2 checks to see if the associated policy exists and if so, deletes and recreates it, along with adding the annotation to the namespace.
```

File: docs/policies.md [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+
+Creation and application of the PSPs is controlled by the presence or absence of certain annotations on the `kube-system` namespace. These map directly to the PSPs which can be created and are:
+
+ * `psp.rke2.io/global-restricted`
+ * `psp.rke2.io/system-unrestricted`
+ * `psp.rke2.io/global-unrestricted`
+
+The following logic is performed at startup for the policies and their annotations:
+
+* If the annotation exists, RKE2 continues without further action
```
Comment (briandowns) [inline docs/policies.md:22]:
```suggestion
* If the annotation exists, RKE2 continues without further action.
```

File: docs/policies.md [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+* If the annotations doesn't exist, RKE2 checks to see if the associated policy exists and if so, deletes and recreates it, along with adding the annotation to the namespace.
+* In the case of the `global-unrestricted-psp`, the policy is not recreated. This is to account for moving between CIS and non-CIS modes without making the cluster less secure.
+* At the time of creating a policy, cluster roles and cluster role bindings are also created to ensure the appropriate policies are put into use by default.
+
+So, after initial startup, operators can modify or delete RKE2's policies and RKE2 will respect those changes. Additionally, to "reset" a policy, an operator just needs to delete the associated annotation from the `kube-system` namespace and restart RKE2.
```
Comment (briandowns) [inline docs/policies.md:27]: Start-up is hyphonated previously. Not sure if you want it here too for consistency.

```suggestion
So, after initial start-up, operators can modify or delete RKE2's policies and RKE2 will respect those changes. Additionally, to "reset" a policy, an operator just needs to delete the associated annotation from the `kube-system` namespace and restart RKE2.
```

File: docs/policies.md [hunk 4]
```diff
@@ -180,7 +199,7 @@ spec:
   - Ingress
 ```
 
-RKE2 applies 2 network policy for all namespaces and updates the namespaces with annotations `default-network-policy` and `default-network-dns-policy`. 
+RKE2 applies 2 network policy for all built-in namespaces and updates the namespaces with annotations `np.rke2.io` and `np.rke2.io/dns`.
```
Comment (briandowns) [inline docs/policies.md:202]: All built-in namespaces get the one policy but only kube-system gets the dns policy.

Comment (cjellick) [reply docs/policies.md:202]: So should this read
```
RKE2 applies the `default-network-policy` policy and `np.rke2.io` annotation to all built-in namespaces. The `kube-system` namespace additionally gets the `default-network-dns-policy` policy and `np.rke2.io/dns` annotation applied to it.
```
?

Comment (briandowns) [reply docs/policies.md:202]: Yeah, that works.

**Review by briandowns — APPROVED** [review]

**Pull request merged**